assets/xml/objects/object_sekihin.xml

```diff
@@ -1,17 +1,18 @@
 ﻿<Root>
+    <!-- Assets for a suns song grave with a kokiri symbol on it -->
     <File Name="object_sekihin" Segment="6">
-        <DList Name="gSunSongKokiriSymbolBaseDL" Offset="0x340" />
-        <DList Name="gSunSongKokiriSymbolDL" Offset="0x4A8" />
-        <DList Name="gSekihinGraveBaseDL" Offset="0x4B8" />
-        <DList Name="gGraveKokiriSymbolDL" Offset="0x5C8" />
+        <DList Name="gSunsSongGraveKokiriSymbolsBaseDL" Offset="0x340" />
+        <DList Name="gSunsSongGraveKokiriSymbolsDL" Offset="0x4A8" />
+        <DList Name="gSunsSongGraveKokiriStoneDL" Offset="0x4B8" />
+        <DList Name="gSunsSongGraveKokiriDL" Offset="0x5C8" />
         <!-- <Blob Name="object_sekihin_Blob_0005E0" Size="0x20" Offset="0x5E0" /> -->
```

Comment:
When I look at this in the C file, I see this:
```c
u8 object_sekihin_unaccounted_0005E0[] = {
    0x99, 0xE0, 0xDE, 0xF7, 0x7B, 0xDF, 0x73, 0x9D, 0x52, 0x95, 0x29, 0x4B, 0x6B, 0x5B, 0x00, 0x01, 
    0x01, 0x23, 0x01, 0x23, 0x01, 0x23, 0x01, 0x23, 0x01, 0x23, 0x01, 0x23, 0x01, 0x23, 0x01, 0x23, 
};
```
That same sequence of bytes can be found in Sekihil too, and when I look where the TLUT is in Sekihig in a hex editor, I see that same sequence of bytes where the TLUT used to reside. In other words, I think these all have the same TLUTs in them, even if they only use one of them. Could you take a look and update the blobs accordingly?

Reply:
Did a little more digging, and I can confirm it's the same TLUT repeated across these three objects. It looks like this (taking sekihig as an example):
```xml
<Texture Name="gSekihigGoronTLUT" OutName="sekihig_goron_tlut" Format="rgba16" Width="4" Height="4" Offset="0x5E0" />
<Texture Name="gSekihigGraveKokiriTLUT" OutName="sekihig_kokiri_tlut" Format="rgba16" Width="4" Height="4" Offset="0x600" />
<Texture Name="gSekihigGraveUnknownTLUT" OutName="sekihig_unknown_tlut" Format="rgba16" Width="4" Height="4" Offset="0x620" />
 <Texture Name="gSekihigGraveTriforceTLUT" OutName="sekihig_triforce_tlut" Format="rgba16" Width="4" Height="4" Offset="0x640" />
```

Not sure how you want to express this with your naming scheme, but this is how the TLUTs are arranged for all of them. The unknown TLUT suggests to me that either there was another grave at some point, or sekihiz was going to use CI textures at some point. Or it could have been something completely different, who knows.

Reply:
Ah I had noticed the same string of bytes, but didn't connect it with the already existing TLUTs. Thanks for spotting that :) .
Also as much as I kind of want to name the `Unknown` one `Zora` (since it seems to fit), I did not as we don't know for sure.